src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -202,7 +201,7 @@
                                         {:view     :share-chat-key
                                          :address  (or compressed-key
                                                        public-key)
-                                         :ens-name preferred-name}])
+                                         :ens-name display-name}])
```

Comment:
after bought first ens name, it will set `preferred-name` rather than `display-name` to ens name, here just used `display-name` instead of `preferred-name`, does this mean we should discard using `preferred-name` ? cc @cammellos @Samyoul

Reply:
I reverted this change. All the updates to how we handle `preferred-name` vs `display-name` vs `primary-name` will be handled in a separate issue/PR.